lib/components/src/blocks/Source.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FunctionComponent } from 'react';
 import { styled, ThemeProvider, convert, themes } from '@storybook/theming';
 import { EmptyBlock } from './EmptyBlock';
 
-import { SyntaxHighlighter } from '../syntaxhighlighter/syntaxhighlighter';
```

Comment:
@shilman investigate perf improvements from `perf/syntax-higlighter-async` branch

Reply:
I believe this was already merged and ended up being the syntax highlighter size reduction PR @ndelangen